src/app/person/person.cpp

```diff
@@ -3,8 +3,8 @@
 #include <numeric>
 #include <vector>
 
-Person::Person(const std::string& name, const std::string& surname, const std::string& pesel)
-    : name_{name}, surname_{surname}
+Person::Person(const std::string& name, const std::string& surname, const std::string& pesel, const Gender& gender)
+    : name_{name}, surname_{surname}, gender_{gender}
 {
     setPesel(pesel);
```

Comment:
Before setting we should perform validation here. Please add validation part and respectively to the output set or not set pesel. 
I suggest using exception here.

Reply:
`setPesel(pesel) { if(validatePesel(pesel)) { pesel_=pesel; } else { throw exception; } }`

Reply:
Done